Source/DafnyLanguageServer.Test/Synchronization/DeclarationLocationMigrationTest.cs

```diff
@@ -279,5 +279,22 @@ var a
       Assert.AreEqual(new Range((6, 6), (6, 7)), yLocation.Name);
       Assert.AreEqual(new Range((6, 6), (6, 7)), yLocation.Declaration);
     }
+
+    [TestMethod]
+    public async Task NullRangeClearsSymbolsTable() {
```

Comment:
Make the test method name describe what the test does and what's the expected outcome.